backend/mcp/client.py

```
@@ -0,0 +1,498 @@
+#!/usr/bin/env python3
+"""
+MCP Client for RadSysX
+
+This module provides a client for interacting with Model Context Protocol servers,
+with a focus on integrating with the RadSysX framework.
+"""
+
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)